src/App.vue

```diff
@@ -1,4 +1,5 @@
-<script setup>
```

Comment:
why this has been changed to lang=js?

Reply:
i'm not sure about this honestly i don't recall where that came from